### ISSUE TITLE
generates an error

### ISSUE THREAD

**Issue by vinayakumarr** [renan-campos/MSA#2]
I installed same verion of numy, scipy, theano,sklearn and drill but still your code generates an error. Please check the attached file for that error
![547b4fd6-2a7f-11e6-99f5-ec5c8da5f841](https://cloud.githubusercontent.com/assets/18217467/15799871/c3e4c0f6-2a88-11e6-8344-a5ac6ba1b295.png)

**Comment by rcampos2029** [comment]
numpy MemoryErrors are typically caused by not having enough contiguous memory to form the array.
[Related Stack Overflow Question](http://stackoverflow.com/questions/6021728/python-numpy-ndarray-shape-limit)

I would suggest trying with a smaller training set to see if that runs without error, then trying to free some resources to run with the larger set.

**Comment by vinayakumarr** [comment]
could you please specify which are all classifiers you are using and its accuracy

**Comment by rcampos2029** [comment]
We used an SVM with a linear kernel, which attained an accuracy of 0.87545.
[Kaggle Competition](https://inclass.kaggle.com/c/sentiment-classification-on-large-movie-review/leaderboard)
More details on the implementation can be found in the paper directory.
